docs/results.md

```diff
@@ -0,0 +1,376 @@
+                └── PerfTestLarge-1572446972804
+```
+
+The results directory name consists of the scenario name (`PerfTestLarge`) and a timestamp (`1572446972804`).
+This directory is generated by Gatling and includes the HTML report, simulation log file, and JSON data files.
+For example:
+```
+PerfTestLarge-1572446972804
+├── PerfTestLarge-1572446972804.csv
+├── PerfTestLarge-1572446972804.csv.html
+├── group_perftestlarge-5163d.html
+├── index.html
+├── js
+├── req_perftestlarge---588ae.html
+├── req_perftestlarge---8453a.html
+├── req_perftestlarge---a1275.html
+├── req_perftestlarge---b4f51.html
+├── req_perftestlarge---db667.html
+├── req_perftestlarge---de2f0.html
+├── simulation.log
+└── style
+
+```
+
+##### HTML report
+The `index.html` file is the Gatling HTML report which will load automatically when this directory is viewed in a web browser.
+For local viewing open the file directly.
+
+##### Simulation log
+The `simulation.log` file is the Gatling simulation log which contains a record of each transaction in the simulation.
+
+##### JSON data
+The JSON data files are contained within the `js` directory. For example:
+```
+PerfTestLarge-1572446972804/js
+├── all_sessions.js
+├── assertions.json
+├── assertions.xml
+├── bootstrap.min.js
+├── gatling.js
+├── global_stats.json
+├── highcharts-more.js
+├── highstock.js
+├── jquery.min.js
+├── menu.js
+├── moment.min.js
+├── stats.js
+├── stats.json
+├── theme.js
+└── unpack.js
+```
+
+The JSON results are contained in the `global_stats.json` and `stats.json` files.
+
+In order to make the relevant test data easily accessible and available for comparison between runs it is extracted into a CSV file via the [`gatling2csv`](https://github.com/puppetlabs/gatling-puppet-load-test/blob/7278352a6626e1f22ae8d17a7d36aa36c94b5cf4/tests/helpers/perf_results_helper.rb#L89) utility.
+This is the `PerfTestLarge-1572446972804.csv` file in the example above.
+As with the other CSV files in the results it is converted to an HTML document via the `csv2html` utility (`PerfTestLarge-1572446972804.csv.html`).
+
+## puppet-metrics-collector
+Metrics data for the test run is extracted from the [`puppet-metrics-collector`](https://github.com/puppetlabs/puppetlabs-puppet_metrics_collector) directory on the 'master' node via the [`collect_metrics_files`](https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/util/metrics/collect_metrics_files.rb) utility.
+This utility uses the timestamps in the `start_epoch` and `end_epoch` files created at the start and end of the run to copy only the data for the duration of the test run.
+The data is extracted into a directory structure mirroring the original source directory and compressed into an archive named with the start and end timestamps: `20191030T144928Z-20191030T184941Z.tar.gz` in the example above.
+The archive is extracted into the `puppet-metrics-collector` directory in the results which contains the relevant data.
+For example:
```

Comment:
I think you want a blank line before this so that it doesn't get folded into the previous paragraph when rendered.